modules/cas_disk/exp_obj.c

```diff
@@ -652,6 +651,17 @@ struct request_queue *casdsk_exp_obj_get_queue(struct casdsk_disk *dsk)
 }
 EXPORT_SYMBOL(casdsk_exp_obj_get_queue);
 
+void casdsk_exp_obj_cleanup_queue(struct casdsk_disk *dsk)
```

Comment:
We can pass `queue` directly to this function instead of `dsk`. Btw this will make this function one-liner, so maybe just call `blk_cleanup_queue()` directly where it's used?

Reply:
done